### PR TITLE
chore: migrate to owl bot

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-containeranalysis/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: owlbot.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 Google LLC
```
Comment (chingor13) [inline owlbot.py:1]: Can we make this a git mv to preserve the license year, or update to the current year?

File: owlbot.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+# Add beta version GrafeasClient to export
+s.replace('src/index.ts',
+        r"""
+export \{v1beta1, v1, ContainerAnalysisClient\};
+export default \{v1beta1, v1, ContainerAnalysisClient\};""",
+        'const GrafeasClient = v1beta1.GrafeasV1Beta1Client;\n' +
+        'type GrafeasClient = v1beta1.GrafeasV1Beta1Client;\n\n' +
+        'export {v1, v1beta1, ContainerAnalysisClient, GrafeasClient};\n' +
+        'export default {v1, v1beta1, ContainerAnalysisClient, GrafeasClient};')
```
Comment (chingor13) [inline owlbot.py:77]: It looks like the `patch_staging` callback is executed once (not per staging directory). Should this replace also go in there?

**Review by chingor13 — CHANGES_REQUESTED** [review]

File: owlbot.py (hunk 2 above)
Comment (SurferJeffAtGoogle) [reply owlbot.py:77]: This replace() is touching a file generated by synthtool's template-generating code.  It's unrelated to the code originating in the staging directory, therefore this is the correct place for it.

File: owlbot.py (hunk 1 above)
Comment (chingor13) [reply owlbot.py:1]: Apparently git does not fully support this so we'll keep the year from the original synth.py.

**Review by chingor13 — APPROVED** [review]

**Pull request merged**